frontend/src/app/modules/tenant-admin/client/client.component.ts

```diff
@@ -87,42 +93,61 @@ export class ClientComponent implements OnInit {
       saveData$ = this.apiService.updateCase(caseDetail, this.type);
     }
 
-    saveData$.subscribe(() => {
+    this.subs.sink = saveData$.subscribe(() => {
       this.snackbarService.success('Persönliche Daten erfolgreich aktualisiert');
       this.router.navigate(['/tenant-admin/clients']);
     });
   }
 
   startTracking(caseDetail: CaseDetailDto) {
-    this.apiService.putApiCall<StartTracking>(caseDetail, 'start-tracking')
+    this.subs.sink = this.apiService.putApiCall<StartTracking>(caseDetail, 'start-tracking')
       .subscribe((data) => {
         this.trackingStart$$.next(data);
         this.tabIndex = 3;
       });
   }
 
   renewTracking(tracking: HalResponse) {
-    this.apiService.putApiCall<StartTracking>(tracking, 'renew')
+    this.subs.sink = this.apiService.putApiCall<StartTracking>(tracking, 'renew')
       .subscribe((data) => {
         this.trackingStart$$.next(data);
         this.tabIndex = 3;
       });
   }
 
   addComment(commentText: string) {
-    this.apiService.addComment(this.caseId, commentText).subscribe((data) => {
+    this.subs.sink = this.apiService.addComment(this.caseId, commentText).subscribe((data) => {
       this.snackbarService.success('Kommentar erfolgreich eingetragen.');
       this.updatedDetail$$.next(data);
     });
   }
```

Comment:
Hier dem entsprechend ein subject zurückgeben, das feuert, sobald von addComment erfolgreich ist. -> Anschließend in checkforClose  in die pipe als switchMap aufnehmen statt map